### ISSUE TITLE
SPIKE: Conformance to WCAG 2.0

### ISSUE THREAD

**Issue by strudelauxpommes** [cosmic-sizing-tool/cosmic-sizing-tool#58]
Basic WCAG 2.0 should be applied for HTML structure.
